lumber-logout.js

```diff
@@ -3,18 +3,15 @@ const os = require('os');
 const fs = require('fs');
 const chalk = require('chalk');
 const logger = require('./services/logger');
+const authenticator = require('./services/authenticator');
 
 program
   .description('Sign out of your account.')
   .parse(process.argv);
 
-const path = `${os.homedir()}/.lumberrc`;
-
-fs.stat(path, (err) => {
-  if (err === null) {
-    fs.unlinkSync(path);
-    console.log(chalk.green('👍  You\'re now unlogged 👍 '));
-  } else if (err.code === 'ENOENT') {
-    logger.error('🔥  You\'re not logged 🔥');
-  }
-});
+(async () => {
+  await authenticator.logout({ log: true })
+    .catch((err) => {
+      console.error(err);
```

Comment:
I'm not sure catching the error here is usefull. The error will still be printed if not catch